test/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -264,7 +265,7 @@ public void adminSignInWithPhone() {
                 .copyOf(PARTICIPANT).withRoles(Sets.newHashSet(Roles.RESEARCHER)).build();
         doReturn(account).when(accountDao).authenticate(study, PHONE_PASSWORD_SIGN_IN);
         doReturn(participant).when(participantService).getParticipant(study, account, false);
-        doReturn(UNCONSENTED_STATUS_MAP).when(consentService).getConsentStatuses(any());
+        doReturn(UNCONSENTED_STATUS_MAP).when(consentService).getConsentStatuses(any(), any());
```

Comment:
In adminSignInWithEmail(), we verify the test by fetching the consent map from the session. Should we do the same in these tests?

* adminSignInWithPhone()
* emailSignInThrowsConsentRequired()
* emailSignInAdminOK()
* phoneSignInThrowsConsentRequired()

Reply:
Added